api/proto/file/v1/file.proto

```diff
@@ -89,4 +94,13 @@ enum FileErr {
     DispatchError = 1; //mandatory
```

Comment:
If you are using the new types of errors (the Enum in a .proto file construction) you can delete these.  They are no longer needed.